translations/fr/get5.phrases.txt

```diff
@@ -296,4 +408,64 @@
     {
         "fr"        "Le veto commencera dans {1} secondes."
     }
+    "SurrenderCommandNotEnabled"
+    {
+        "fr"        "La commande de capitulation n'est pas acctivée."
+    }
+    "SurrenderMinimumRoundDeficit"
+    {
+        "fr"        "Vous devez avoir terminé {1} manche(s) pour pouvoir capituler."
+    }
+    "SurrenderInitiated"
+    {
+        "fr"        "A vote to surrender was initiated by {1}. Your team must reach {2} votes within {3} seconds."
+    }
+    "SurrenderVoteStatus"
+    {
+        "fr"        "{1} des {2} votes de capitulation requis ont été exprimés."
+    }
+    "SurrenderSuccessful"
+    {
+        "fr"        "{1} a capitulé."
```

Comment:
{1} à abandonner.

Reply:
Currently says `{1} a abandonné.` @Gryfenfer97 disagree?

Reply:
Yes it's "abandonné"